hadoop-hdds/docs/content/interface/OzoneFS.md

```diff
@@ -23,9 +23,9 @@ summary: Hadoop Compatible file system allows any application that expects an HD
 
 The Hadoop compatible file system interface allows storage backends like Ozone
 to be easily integrated into Hadoop eco-system.  Ozone file system is an
-Hadoop compatible file system.
+Hadoop compatible file system. Currently, Ozone supports two scheme: o3fs and ofs.
```

Comment:
Can we add some highlights for the difference between the two, like you have mentioned in Line 147.

Reply:
The main difference between the two has been added.